var/spack/repos/spack_repo/builtin/packages/py_vesin/package.py

```diff
@@ -0,0 +1,48 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyVesin(PythonPackage):
+    """Computing neighbor lists for atomistic system."""
+
+    homepage = "https://luthaf.fr/vesin/latest/index.html"
+    pypi = "vesin/vesin-0.3.7.tar.gz"
+
+    import_modules = ["vesin"]
+
+    maintainers("HaoZeke", "luthaf")
+    license("BSD-3-Clause", checked_by="HaoZeke")
+
+    version("0.3.7", sha256="52c11ac0ba775c228f06779877cf8641854edab7ea59036093ef5e8447379de0")
+    version("0.3.6", sha256="70f1af8483f6197a009ffa425ed424ecde050187d5a07e83cb811025f62adfc9")
+    version("0.3.5", sha256="1ad9a6bcd44d4313e8e2ce81ed58aa23f37e7d4cebd9570323c98086a3c41902")
```

Comment:
These two versions are broken and yanked from PyPI, so maybe not worth having here ^_^